ui/components/app/modals/convert-token-to-nft-modal/convert-token-to-nft-modal.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import Modal from '../../modal';
+import Typography from '../../../ui/typography';
+import {
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  ALIGN_ITEMS,
+  DISPLAY,
+} from '../../../../helpers/constants/design-system';
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { ADD_COLLECTIBLE_ROUTE } from '../../../../helpers/constants/routes';
+
+const ConvertTokenToNFTModal = ({ hideModal, tokenAddress }) => {
+  const history = useHistory();
+  const t = useI18nContext();
+  return (
+    <Modal
+      onSubmit={() => {
+        history.push({
+          pathname: ADD_COLLECTIBLE_ROUTE,
+          state: { tokenAddress },
+        });
+        hideModal();
+      }}
+      submitText={t('yes')}
+      onCancel={() => hideModal()}
+      cancelText={t('cancel')}
+    >
+      <div className="convert-token-to-nft-modal">
+        <Typography
+          variant={TYPOGRAPHY.H6}
+          fontWeight={FONT_WEIGHT.NORMAL}
+          boxProps={{
+            marginTop: 2,
+            display: DISPLAY.INLINE_FLEX,
+            alignItems: ALIGN_ITEMS.CENTER,
```

Comment:
Not sure these layout props are necessary?

```suggestion
```

Reply:
removed here: https://github.com/MetaMask/metamask-extension/pull/13291/commits/71a02813c79dbb632742095c0ea0566fd029eb39